CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ enable_testing()
 set(CMAKE_C_VISIBILITY_PRESET hidden)
 set(CMAKE_CXX_STANDARD 11)
 
+set(CMAKE_INSTALL_LIBDIR lib CACHE PATH "Installation directory for libraries")
```

Comment:
@compnerd Please comment on the PR and not the commit as force pushing removes them.

This is needed because its value is `lib/x86-gnu-linux/` (or similar), where as we need to install the libraries at `lib/`

Reply:
`GNUInstallDirs` will set the value to `lib` or `lib64` depending on the system that you are building for.  The only place that the `CMAKE_INSTALL_LIBDIR` uses the form that you specify is when you are on Debian as that is the style that Debian uses.  You should be able to override that from the command line.  I think that is better handled in `build-script` rather than hardcoding it here.

See https://cmake.org/cmake/help/v3.4/module/GNUInstallDirs.html

Reply:
I see. I don't know why it sets that path for Ubuntu then. Sure, I can override it in the build-script if that is preferable.

Reply:
Alright, done!